cmd/aries-agentd/startcmd/start.go

```diff
@@ -117,38 +136,48 @@ func Cmd(server server) (*cobra.Command, error) {
 		return nil, fmt.Errorf("tried to mark DB path flag as required but it was not found: %s", err)
 	}
 
+	startCmd.Flags().StringSliceVarP(&webhookURLs, AgentWebhookFlagName, AgentWebhookFlagShorthand, []string{},
+		AgentWebhookFlagUsage)
+	err = startCmd.MarkFlagRequired(AgentWebhookFlagName)
+	if err != nil {
+		return nil, fmt.Errorf("tried to mark agent webhook host flag as required but it was not found: %s", err)
+	}
+
 	return startCmd, nil
 }
 
-func startAgent(server server, host, inboundHost, dbPath string) error {
-	if host == "" {
+func startAgent(parameters *agentParameters) error {
+	if parameters.host == "" {
```

Comment:
May be unrelated to your current change.
Nice to have var error for (MissingHostErrorMessage string), than doing errors.new + to lower everytime

Reply:
Done!